scripts/test_helpers.bats

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bats
 
-# Allow to run the tests locally provided that bats-helpers are installed in $HOME/bats-core
-bats_helpers_root="${HOME}/bats-core"
+# Allow to run the tests locally provided that bats-helpers are installed.
+# If $BATS_HELPERS_ROOT is set, an attempt is made to load them from that directory,
```

Comment:
```suggestion
# If $BATS_CORE_ROOT is set, an attempt is made to load them from that directory,
```

Reply:
I figured I would do it this way to reduce scope creep, i.e. keep rest unchanged.

Reply:
I thought that's a typo in the comment, because below you check if `BATS_CORE_ROOT` exists

Reply:
d'oh, thanks.

Reply:
fixed